pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -14,9 +14,15 @@
 #include <sys/types.h>
 #include <errno.h>
 
+#include "bootstrap_name.h"
+#ifndef HAS_NO_SDL_BOOTSTRAP
```

Comment:
Since this includes every bootstrap but SDL2, would it be simpler to just check for BOOTSTRAP_NAME_SDL2, or are do you have something in mind for this?

Reply:
I just named this badly, it's supposed to mean "this bootstrap uses SDL headers". I think at least one other (might have been pygame) uses SDL headers as well. Should rename it for clarity, good catch

Reply:
I thought that was the intention, but it looks like your pygame file includes "#define HAS_NO_SDL_BOOTSTRAP", is that a mistake?

My instinct would be to just hardcode `if defined(BOOTSTRAP_NAME_PYGAME) || defined(BOOTSTRAP_NAME_SDL2)`, and revisit it if in fact we add another SDL bootstrap. Especially because I'd consider pygame/sdl1 and sdl2 to be so different as to be different libraries that happen to have a header file with the same name.

I don't feel that strongly about it though, it's simple enough to change if we do something in the future that would encourage a different scheme.

Reply:
Nice, this was actually a legitimate mistake. This was defined for `pygame` when it shouldn't have been, I checked back on the pre-merge source. Fixed now, and renamed to `BOOTSTRAP_USES_NO_SDL_HEADERS`. :+1: